File: python3/268/Solution.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        length_of_list = len(nums)

        expected_sum = length_of_list * (length_of_list + 1) // 2
        actual_sum = sum(nums)

        return expected_sum - actual_sum

File: java/268/Solution.java
import java.util.stream.IntStream;

class Solution {
    public int missingNumber(int[] nums) {
        int lengthOfArray = nums.length;

        int expectedSum = lengthOfArray * (lengthOfArray + 1) / 2;
        int actualSum = IntStream.of(nums).sum();

        return expectedSum - actualSum;
    }
}
